ngraph/core/src/builder/reshape.cpp

```diff
@@ -202,3 +202,52 @@ shared_ptr<Node> builder::opset1::squeeze(const Output<Node>& value, vector<size
     }
     return builder::opset1::reshape(value, output_shape);
 }
+
+shared_ptr<Node> builder::opset1::collapse(const Output<Node>& value,
+                                           const size_t start_axis,
+                                           const size_t end_axis)
+{
+    if (start_axis == end_axis)
+    {
+        return value.get_node_shared_ptr();
+    }
+
+    if (value.get_partial_shape().is_static())
+    {
+        auto shape = value.get_shape();
+        size_t collapsed_axis_size = accumulate(next(begin(shape), start_axis),
```

Comment:
I'd like to see a short comment explaining what gets multiplied here. Can you add a single sentence above this line?

Reply:
Done